trac_fedmsg_plugin.py

```diff
@@ -106,6 +112,20 @@ def publish(self, topic, **msg):
         msg['agent'] = currently_logged_in_user()
         fedmsg.publish(modname='trac', topic=topic, msg=msg)
 
+    def attachment_added(self, attachment):
+        self.publish(topic='attachment.added',
+                     attachment=attachment2dict(attachment))
+
+    def attachment_deleted(self, attachment):
+        self.publish(topic='attachment.deleted',
+                     attachment=attachment2dict(attachment))
+
+    def attachment_reparented(self, attachment, old_parent_realm, old_parent_id):
+        self.publish(topic='attachment.reparented',
```

Comment:
That means that the attachment was moved between parents. So for example from Wiki/Whatever to ticket/3